lib/src/contentsteering/callContentSteeringServer.ts

```diff
@@ -0,0 +1,49 @@
+interface ContentSteeringServerResponse {
+	status: number;
+	data: string;
+	headers: Record<string, string>;
+	url: string;
+}
+
+export async function callContentSteeringServer(
+	serverURI: string | null,
+	onSuccess: (response: ContentSteeringServerResponse) => void,
+	onError: (response: ContentSteeringServerResponse) => void,
+	onTimeout: (serverURI: string) => void,
+	timeout: number = 5000
```

Comment:
Maybe these functions should come in an object named callbacks just like it's implemented in hls.js:
<img width="585" alt="Screenshot 2024-03-21 at 9 26 11 AM" src="https://github.com/qualabs/common-media-library/assets/121059628/d1f687a1-9c49-46f1-a68e-19eea289c062">
On the other hand in dash it also comes inside an object: 
<img width="1031" alt="image" src="https://github.com/qualabs/common-media-library/assets/121059628/12b23674-f929-4964-b280-b2d642480a58">
What do you think?